libvncserver/rfbserver.c

```diff
@@ -358,6 +358,7 @@ rfbNewTCPOrUDPClient(rfbScreenInfoPtr rfbScreen,
       rfbReleaseClientIterator(iterator);
       rfbLog("  %lu other clients\n", (unsigned long) otherClientsCount);
 
+#ifndef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
Actually, where does this one get defined?

Reply:
This is defined by oss-fuzz in `CFLAGS` like `export CFLAGS=-D FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION`

I think the bigger question is below...

Reply:
Alright